core/server/api_container/server/startosis_engine/kurtosis_instruction/add_service/add_service_shared.go

```diff
@@ -85,6 +85,16 @@ func validateSingleService(validatorEnvironment *startosis_validator.ValidatorEn
 	return nil
 }
 
+func invalidServiceNameErrorText(
+	serviceName service.ServiceName,
+) string {
+	return fmt.Sprintf(
+		"Service name '%v' is invalid as it contains disallowed characters. Service names must adhere to the RFC 1123 standard, specifically implementing this regex and be 1-63 characters long: %s. This means the service name must only contain lowercase alphanumeric characters or '-', and must start and end with a lowercase alphanumeric character.",
```

Comment:
nit: Wondering if it's worth printing the regexp here, especially as you're providing the details already. Maybe hiding the regexp would be better. No strong opinion though